src/methods/method_overloading/medium/q4/AveOfThree.java

```diff
@@ -0,0 +1,15 @@
+package methods.method_overloading.medium.q4;
+
+/*
+ Overload a Java Method so that it can take three integer variables or three double variables and returns the average
```

Comment:
```suggestion
 Overload a Java Method called average so that it can take three integer variables or three double variables and returns the average
```